Gopkg.lock

```diff
@@ -9,15 +9,9 @@
 
 [[projects]]
   branch = "master"
-  name = "github.com/Azure/go-ansiterm"
```

Comment:
@grahamwhaley this is not a bug, this is because we removed recently a part of code relying on this docker package. And when you ask dep to do the dep ensure, it first try to make sure that everything is right on the repo. Thus, we are using this update of toml to update properly our repo vendoring.